components/esm/loadland.cpp

```diff
@@ -175,6 +175,48 @@ namespace ESM
 
     }
 
+    void Land::blank()
+    {
+        if (mLandData)
+        {
+            delete mLandData;
```

Comment:
Best to set mLandData to nullptr here so we avoid a dangling pointer in case of an exception.

Reply:
Also you don't need to check for null before calling ```delete```. Like ```free```, it's a no-op to delete null pointers (some code checkers will complain about a redundant null check).

Reply:
Looking at how it is used, I don't even need to delete it. If it exists then I can reuse it.